Makefile

```diff
@@ -0,0 +1,6 @@
+SHELL := /bin/bash
+all:
+	sudo docker build --network host -t pka .
+	sudo docker image tag pka:latest localhost:5000/pka:latest
+	sudo docker image push localhost:5000/pka:latest
+
```

Comment:
Please do not commit this file or other artifacts tied to your development environment.

Reply:
removed